index.html

```diff
@@ -0,0 +1,228 @@
+                    </div>
+                    <p class="course-item-description">in this section of the
+                        program we talk
+                        about back-end development in general and in ruby on
+                        rails especially.
+                    </p>
+                </li>
+            </ul>
+            <a href="#">SEE THE WHOLE PROGRAM</a>
+            <button>Join Our 2021 Webinar</button>
+        </section>
+
+        <section id="featured-speakers">
+            <h2 class="text-center">Featuerd Speakers</h2>
+            <div id="hr-wrapper">
+                <hr>
+            </div>
+            <ul>
+                <li>
+                    <img src="images/speaker-1.png" alt="speaker number one">
+                    <div>
+                        <h3 class="bold speaker-name"></h3>
+                        <p class="orange-color bold speaker-job"><p>
+                                <hr class="speaker-hr">
+                                <p class="speaker-graduated"></p>
+                            </div>
+                        </li>
+                        <li>
+                            <img src="images/speaker-2.png" alt="speaker number
+                                one">
+                            <div>
+                                <h3 class="bold speaker-name"></h3>
+                                <p class="orange-color bold speaker-job"></p>
+                                <hr class="speaker-hr">
+                                <p class="speaker-graduated"></p>
+                            </div>
+                        </li>
+                        <li class="hide-on-mobile">
+                            <img src="images/speaker-3.png" alt="speaker number
+                                one">
+                            <div>
+                                <h3 class="bold speaker-name"></h3>
+                                <p class="orange-color bold speaker-job"></p>
+                                <hr class="speaker-hr">
+                                <p class="speaker-graduated"></p>
+                            </div>
+                        </li>
+                        <li class="hide-on-mobile">
+                            <img src="images/speaker-4.png" alt="speaker number
+                                one">
+                            <div>
+                                <h3 class="bold speaker-name"></h3>
+                                <p class="orange-color bold speaker-job"></p>
+                                <hr class="speaker-hr">
+                                <p class="speaker-graduated"></p>
+                            </div>
+                        </li>
+                    </ul>
+                    <div id="more-button-wrapper">
+                        <button>More <i class="bi bi-chevron-down orange-color"></i></button>
+                    </div>
+                </section>
+
+                <section id="partners">
```

Comment:
- [ ] Your _partners_ section does not match the provided template design. Kindly ensure to add the necessary icons and adjust the spacing of this section.

**Your partners section:**
![image](https://user-images.githubusercontent.com/12745474/132547898-0550a264-cd28-4131-aec1-28f8002be76f.png)

**The original partners section:**
![image](https://user-images.githubusercontent.com/12745474/132548028-d89be19c-0432-4fd7-8d51-1feedb51787e.png)